auacm/app/static/js/controllers/EditProblemController.js

```diff
@@ -65,6 +75,8 @@ app.controller('EditProblemController', ['$scope', '$route', '$http',
             fd.append('sol_file', $scope.solFile);
         if (typeof prob.appeared !== 'undefined')
             fd.append('appeared_in', prob.appeared);
+        if (typeof prob.comp_release !== 'undefined');
+            fd.append('comp_release', parseInt(prob.comp_release));
```

Comment:
It appears that we're not actually using the `comp_release` field on the server side, so problems that are added via the dropdown are never actually updated.